zadani/soft/style.css

```diff
@@ -15,29 +16,311 @@ body {
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    
 }
 
 
 /* HLAVICKA */
+.hlavicka {    
+    background-image: linear-gradient(rgba(0,0,0,0.3),rgba(0,0,0,0.3)), 
+                    url("../../images/island-hlavicka.jpg"); 
+    background-position: center;
+    background-size: cover;
+    min-height: 200px;
+    text-align: center;
+    padding: 5px;
+    margin: 0
+}
 
 .titulek {
-    font-family: 'Raleway', serif;
+    font-size: 50px;
+    font-family:'Raleway', serif;
+    color:white;
+    padding:20px;
```

Comment:
Tady by možná bylo fajn nastavit společný padding ze stran pro celou hlavičku a u jednotlivých prvků už jen řešit odsazení v ose x - zespodu a zeshora.